prime-angular-frontend/src/app/modules/adjudication/pages/enrollee-toa-compare-page/enrollee-toa-compare-page.component.ts

```diff
@@ -0,0 +1,62 @@
+import { AdjudicationRoutes } from '@adjudication/adjudication.routes';
+import { AdjudicationResource } from '@adjudication/shared/services/adjudication-resource.service';
+import { Component, OnInit, ViewEncapsulation } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+
+import { Subscription } from 'rxjs';
+
+import { RouteUtils } from '@lib/utils/route-utils.class';
+import { AgreementTypeGroup } from '@shared/enums/agreement-type-group.enum';
+import { AgreementType } from '@shared/enums/agreement-type.enum';
+import { AgreementVersion } from '@shared/models/agreement-version.model';
+import { FormatDatePipe } from '@shared/pipes/format-date.pipe';
+
+@Component({
+  selector: 'app-enrollee-toa-compare-page',
+  templateUrl: './enrollee-toa-compare-page.component.html',
+  styleUrls: ['./enrollee-toa-compare-page.component.scss'],
+  encapsulation: ViewEncapsulation.None
```

Comment:
You don't want to use ViewEncapsulation.None even though it's an option. Components should remain self contained.